packages/rich-text/src/plugins/Marks/Subscript.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+import { SubscriptIcon } from '@contentful/f36-icons';
+import { createSubscriptPlugin as createDefaultSubscriptPlugin } from '@udecode/plate-basic-marks';
+import { MARKS } from '@contentful/rich-text-types';
+import * as Slate from 'slate-react';
+import { someHtmlElement } from '@udecode/plate-core';
+
+import { RichTextPlugin } from '../../types';
+import { createMarkToolbarButton } from './components/MarkToolbarButton';
+import { buildMarkEventHandler } from './helpers';
+
+export const ToolbarSubscriptButton = createMarkToolbarButton({
+  title: 'Subscript',
+  mark: MARKS.SUBSCRIPT,
+  icon: <SubscriptIcon viewBox="0 0 23 18" />,
+});
+
+export function Subscript(props: Slate.RenderLeafProps) {
+  return <sub {...props.attributes}>{props.children}</sub>;
+}
+
+export const createSubscriptPlugin = (): RichTextPlugin =>
+  createDefaultSubscriptPlugin({
+    type: MARKS.SUBSCRIPT,
+    component: Subscript,
+    handlers: {
+      onKeyDown: buildMarkEventHandler(MARKS.SUBSCRIPT),
+    },
+    deserializeHtml: {
+      rules: [
+        {
+          validNodeName: ['SUB'],
+        },
+      ],
+      query: (el) => {
+        return !someHtmlElement(el, (node) => node.style.fontWeight === 'normal');
+      },
```

Comment:
You don't need this AFAIK. It was only needed for `bold` because it could be inferred from the fontWeight.

Reply:
You are right this is not needed